src/pint/fermi_toas.py

```diff
@@ -137,7 +158,7 @@ def load_Fermi_TOAs(
             weights = weights[idx]
 
     # limit the TOAs to ones in selected MJD range
-    mjds_float = np.array([r[0] + r[1] for r in mjds])
+    mjds_float = np.asarray([r[0] + r[1] for r in mjds])
```

Comment:
Why this change? The input is always a list so you'll always wind up with a new array.

Reply:
To me, there's no downside to using asarray, even though they are indeed functionally identical here.